fdbclient/ServerKnobs.cpp

```diff
@@ -411,6 +411,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( ROCKSDB_COMPACTION_READAHEAD_SIZE,                   32768 ); // 32 KB, performs bigger reads when doing compaction.
 	init( ROCKSDB_BLOCK_SIZE,                                  32768 ); // 32 KB, size of the block in rocksdb cache.
  	init( ENABLE_SHARDED_ROCKSDB,                              false );
+ 	init( MAX_PHYSICAL_SHARD_BYTES,                        500000000 );
```

Comment:
Add a comment of a readable size, e.g., 500MB. Is there any reason 500MB is chosen?

Reply:
No. Just choose a random number here. When we want to host TBs data, we may need to set it as GBs.